pkgs/development/libraries/soci/default.nix

```
@@ -29,7 +30,13 @@ stdenv.mkDerivation rec {
   # Do not build static libraries
   cmakeFlags = [ "-DSOCI_STATIC=OFF" "-DCMAKE_CXX_STANDARD=11" "-DSOCI_TESTS=off" ];
 
-  nativeBuildInputs = [ cmake ];
+  nativeBuildInputs = [
+    cmake
+  ] ++ lib.optionals stdenv.isDarwin (
+    (with darwin.apple_sdk_11_0.frameworks; [
+      Kerberos
+    ])
+  );
```

```suggestion
  ] ++ lib.optionals stdenv.isDarwin [
    darwin.apple_sdk_11_0.frameworks.Kerberos
  ];
```

https://nix.dev/guides/best-practices#with-scopes

---

thanks! sorry didn't realize I got logged out from GitHub and could not accept. Hence made [17ba915](https://github.com/NixOS/nixpkgs/pull/275884/commits/17ba91535836c1fa7877dd7997ca163329e907b9).